src/ValueFormatters/QuantityFormatter.php

```diff
@@ -191,13 +191,13 @@ private function getRoundingExponent( QuantityValue $quantity ) {
 	}
 
 	/**
-	 * @param DecimalValue $margin
+	 * @param DecimalValue|null $margin
 	 * @param int $roundingExponent
 	 *
 	 * @return string|null Text
 	 */
-	private function formatMargin( DecimalValue $margin, $roundingExponent ) {
-		if ( $this->options->getOption( self::OPT_SHOW_UNCERTAINTY_MARGIN ) ) {
+	private function formatMargin( DecimalValue $margin = null, $roundingExponent ) {
+		if ( $margin !== null && $this->options->getOption( self::OPT_SHOW_UNCERTAINTY_MARGIN ) ) {
```

Comment:
IF_NOT_ZERO means IF_KNOWN_AND_NOT_ZERO, of course. basically, we'll want an option to show bounds when given (including zero), or to always hide the bounds if they are zero (current behavior).